Gruntfile.js

```diff
@@ -183,7 +183,8 @@ var _              = require('lodash'),
                 // #### All Unit tests
                 unit: {
                     src: [
-                        'core/test/unit/**/*_spec.js'
+                        'core/test/unit/**/*_spec.js',
+                        'core/server/apps/private-blogging/tests/*_spec.js'
```

Comment:
Would it make sense to make it future proof with: `'core/server/apps/**/tests/*_spec.js'`?